src/jit/simd.cpp

```diff
@@ -346,7 +346,8 @@ var_types Compiler::getBaseTypeAndSizeOfSIMDType(CORINFO_CLASS_HANDLE typeHnd, u
             size = getSIMDVectorRegisterByteLength();
         }
 
-        *sizeBytes = size;
+        *sizeBytes    = size;
+        foundSIMDType = true;
```

Comment:
This assignment changes the compiler instances state.
use a call here instead:

usesSIMDTypes();